plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -205,48 +223,64 @@ struct FileIndex {
                 if (meta.onClick.length != 0)
                     setAttribute("onclick", meta.onClick);
             }
+
             foreach (m; s.muts) {
                 if (!ids.contains(m.id)) {
                     ids.add(m.id);
                     muts.put(MData(m.id, m.txt, m.mut, db.getMutantationMetaData(m.id)));
-                    const inside_fly = format(`%-(%s %)`, s.muts.map!(a => styleHover(m.id, a)))
-                        .toJson;
-                    const fly = format(`fly(event, %s)`, inside_fly);
                     with (d0.addChild("span", m.mutation)) {
                         addClass("mutant");
                         addClass(s.tok.toName);
                         setAttribute("id", m.id.to!string);
-                        setAttribute("onmouseenter", fly);
-                        setAttribute("onmouseleave", fly);
                     }
                     d0.addChild("a").setAttribute("href", "#" ~ m.id.to!string);
+                    auto testCases = db.getTestCases(m.id);
+                    testCases.sort!(sort_tcs_on_kills);
```

Comment:
Mmm this will result in many database queries. With the "fix" of a specialized DB query the testCases will already be sorted.
future improvement :)